get_trust_anchor.py

```diff
@@ -385,12 +385,15 @@ def main():
     trust_anchor_filename = "root-anchors.xml"
     signature_filename = "root-anchors.p7s"
     icann_ca_filename = "icanncacert.pem"
+    temp_files = [trust_anchor_filename, signature_filename, icann_ca_filename]
```

Comment:
If they say "keep", we can use the static files. If not, we'll use temporary files instead of writing to CWD?